### ISSUE TITLE
unable to set Library Path on NFS

### ISSUE THREAD

**Issue by azrin1972** [scakemyer/plugin.video.quasar#419]
Hi, I've been using Quasar since 0.9.23 and it has been an amazing ad-ons. Kudos to you and your team.

Today I'm trying to set the library path to my NAS using NFS and I always get "Library path not set" error.

here is the log file

[kodi.txt](https://github.com/scakemyer/plugin.video.quasar/files/179010/kodi.txt)

**Comment by scakemyer** [comment]
From the README:
> Can I set it to download directly to my NAS?
Yes, but you need to mount your NAS via the OS, not via Kodi.

**Comment by azrin1972** [comment]
@scakemyer thanks fro the note. Download is working fine, only library path that gives me the issue.

**Comment by scakemyer** [comment]
Doesn't look like download is working fine:
```
10:45:00 T:1908265560  NOTICE: [plugin.video.quasar] Deleting stale files set([u'Torrents'])
...
10:45:01 T:1908265560  NOTICE: [plugin.video.quasar] ERRO  btservice    ▶ NewBTService     Unable to create Torrents folder
```
You seem to insist on using your NFS shares directly, when you need to mount them at the OS level, and it was mentioned many times how that might not work well for downloads.

**Comment by azrin1972** [comment]
I'm not sure what you mean it does not work, but I've been using it since Pulsar days at I have no issues. I can have the files if I did not set it to delete after watching. so After watching I just moved the file to my archive. It is a great thing to be able to download straight to my NAS via NFS. I love it!!

The reason why I'm trying to set the library path to NAS is I have a few Android box, so sharing the same library will sync my TV Shows and movies across all the boxes. Anyhow, I'm still happy with what is available today. Great job @scakemyer 

By the way, I cannot find a way to mount the nfs on my android box - but that's my issues not yours :)
